andela/ah-infinity-stones-frontend
# #161966873 Implement Social Login Feature

#### What does this PR do?
- Implement social login.

#### Description of Task to be completed?
- Implement login with facebook, twitter and google.
- Update the mockups
- Write tests

#### How should this be manually tested?
1. `$ git clone -b ft-social-login-161966873 https://github.com/andela/ah-infinity-stones-frontend.git`  and `$ cd ah-infinity-stones-frontend`
2. Install the dependencies - `$ npm install`
3. Start the app by running `$ npm start` - This will run the app in the development mode.  Open http://localhost:3000/login  to view it in the browser.
The page will reload if you make edits. 	You will also see any lint errors in the console.
4. Test the app by running `$ npm test` - This will Launch the test runner in the interactive watch mode.

#### What are the relevant pivotal tracker stories?
[#161966873](https://www.pivotaltracker.com/story/show/161966873)

#### Screenshots
![image](https://user-images.githubusercontent.com/17976289/51376430-5a9ece00-1b19-11e9-9306-d9fc2af13c5a.png)
![image](https://user-images.githubusercontent.com/17976289/51376563-afdadf80-1b19-11e9-84cb-3919c0d8c42e.png)
![image](https://user-images.githubusercontent.com/17976289/51376652-e44e9b80-1b19-11e9-85d7-4d6f7cdbb765.png)
![image](https://user-images.githubusercontent.com/17976289/51376752-2546b000-1b1a-11e9-94fe-aea05bdbfd5b.png)
![image](https://user-images.githubusercontent.com/17976289/51376793-414a5180-1b1a-11e9-8a2a-9c4a61666b63.png)
